var/spack/repos/builtin/packages/ecflow/package.py

```diff
@@ -24,6 +24,8 @@ class Ecflow(CMakePackage):
     maintainers("climbfuji", "AlexanderRichert-NOAA")
 
     # https://confluence.ecmwf.int/download/attachments/8650755/ecFlow-5.8.3-Source.tar.gz?api=v2
```

Comment:
Can you remove this unnecessary comment please?

Reply:
Sure!